### PR TITLE
🌿 Removing old BHS Code

### PR DESCRIPTION
**Pull request by CEbbinghaus**
`This PR is to quickly remove any remaining BHS code that has a little remaining impact on the app in its current state to cut down on the amount of code that needs refactoring.

These changes would hopefully also cull any unused remains and further improve the codebase by making up for some tech debt.

### CODE REVIEW

**Comment by CEbbinghaus** [comment]
Remaining Features before it can be merged:
- [ ] Validator for Form submission to ensure Galaxy is valid

File: public/commonUI.js [hunk 1]
```diff
@@ -1220,14 +1187,10 @@ blackHoleSuns.prototype.buildMenu = function (
...
-        list = list.sort((a, b) =>
-            a.name.toLowerCase() > b.name.toLowerCase()
-                ? 1
-                : a.name.toLowerCase() < b.name.toLowerCase()
-                ? -1
-                : 0
-        );
+    
+    if (options.sort){
+        list = list.sort((a, b) => a.name.localeCompare(b.name));
```
Comment (CEbbinghaus) [inline public/commonUI.js:1192]: Neat little shorthand for sorting an array of strings

File: public/nmsce.js [hunk 2]
```diff
@@ -1487,7 +1513,7 @@ class NMSCE {
     }
 
     openSearch() {
-        window.open("this.html?s=" + this.last.addr.nameToId() + "&g=" + this.last.galaxy.nameToId(), '_self')
+        window.open("bhs?s=" + this.last.addr.nameToId() + "&g=" + this.last.galaxy.nameToId(), '_self')
```
Comment (CEbbinghaus) [inline public/nmsce.js:1516]: This is not correct. neither `this.html` nor `bhs` will resolve. Has to be changed before it gets merged

File: public/tmputil.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+    `;
+    
+    let locElement = JElement.find("#id-" + id);
+
+    locElement.empty();
+    locElement.append(`<div class="row">${titleHtml}${inputHtml}<div>`);
+    let el = locElement.find(`#btn-${id}`);
+
+    window.test = () => el.get(0).setCustomValidity("Invalid field.");
```
Comment (CEbbinghaus) [inline public/tmputil.js:42]: 🎗️ Testing code that is left over. Has to be removed before it gets merged

**Review by CEbbinghaus — COMMENTED** [review]
Left a comment for you spip but I noted 2 things i have to change before it can be merged

File: public/index.html [hunk 4]
```diff
@@ -2,16 +2,29 @@
 <html lang="en">
 
 <head>
-    <title>No Man's Sky Black Hole Suns</title>
-    <meta name="description" content="Player addon for No Man's Sky">
-    <meta name="subject" content="Player addon for the game No Man's Sky.">
+    <title>No Man's Sky Coordinate Exchange (NMSCE)</title>
```
[the remark this answers is not in the document]
Comment (CEbbinghaus) [reply public/index.html:5]: I realized that these aren't necessarily my changes but could be yours from when you wrote nmsce. In that case it should be good

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by spip01 — APPROVED** [review]

**Review by CEbbinghaus — COMMENTED** [review]
I can't see anything wrong with this

**Pull request merged**